packages/nx/src/command-line/release/command-object.ts

```diff
@@ -115,8 +115,8 @@ export const yargsReleaseCommand: CommandModule<
           );
         }
         const nxJson = readNxJson();
-        if (argv.groups?.length) {
-          for (const group of argv.groups) {
+        if ((argv.groups as string[] | string)?.length) {
+          for (const group of argv.groups as string[] | string) {
             if (!nxJson.release?.groups?.[group]) {
```

Comment:
Type checks

Reply:
These are not type checks, they are the opposite. They are type assertions, where you are telling the compiler what the type is. Please revert I'm not sure why you are touching release command object in a PR regarding bun

Reply:
This is for the green check....

Reply:
This again must be something unusual about your setup. There is no known general issue with this code on master, nor on mine or other's machines. Please revert this change and let's see what happens in CI

Reply:
Either that or please link me to a CI run where this failed, I cannot see one